packages/react-native-editor/ios/GutenbergDemo/GutenbergViewController.swift

```diff
@@ -422,6 +425,29 @@ extension GutenbergViewController: GutenbergBridgeDataSource {
     func gutenbergMediaSources() -> [Gutenberg.MediaSource] {
         return [.filesApp, .otherApps]
     }
+    
+    private func isUITesting() -> Bool {
+        guard ProcessInfo.processInfo.arguments.count >= 2 else {
+            return false
+        }
+        return ProcessInfo.processInfo.arguments[1] == "uitesting"
```

Comment:
`uitesting` is used [here](https://github.com/WordPress/gutenberg/blob/0f0890b99c213ae91969b35dc555b9c6a0363851/packages/react-native-editor/ios/GutenbergDemo/AppDelegate.swift#L23-L28) to tweak the app for testing.